server/src/main/kotlin/core/gameflow/showdown.kt

```diff
@@ -4,3 +4,42 @@ sealed class ShowdownAction
 
 data class ShowCards(val playerId: Int) : ShowdownAction()
 data class MuckCards(val playerId: Int) : ShowdownAction()
+
+fun resolveShowdown(handState: HandState): List<ShowdownAction> {
+
+    val orderedPlayers = when {
+        handState.lastAggressor == null -> handState.orderedPlayers(handState.buttonPosition + 1)
```

Comment:
There is rule that says that when no bet is made(e.g on the River), players should show their cards in a normal betting order?

Reply:
This rule applies when no bet is made on the river. Previous betting rounds don't matter, if everyone checks on the last betting round, showdown order is the same as betting order.

Reply:
ok